### PR TITLE
Feature 151 context namespace

### LINKED ISSUE #151: context must be a map
So I have been running into this with @smrgeoinfo and I saw it in the example by @datadavev 

Using Dave's example of 

```JSON
{
  "@context":"https://schema.org/",
  "@type":"Dataset",
  "name":"test",
  "description": "This is a description of the test. Here's some more words to make it long enough."
}
```

If you place this in the JSON-LD playground [link](https://tinyurl.com/ybsn7ply) you will see it expands to http, not https

modify the context to a map as 

```JSON
{
  "@context": {
    "@vocab": "https://schema.org/"
  },
  "@type": "Dataset",
  "name": "test",
  "description": "This is a description of the test. Here's some more words to make it long enough."
}
```

It will expand correctly with https  as at https://tinyurl.com/y99kj7d7

reference https://www.w3.org/TR/json-ld/#context-definitions

specifically:

```
A context definition MUST be a map whose keys MUST be either terms, 
compact IRIs, IRIs, or one of the keywords @base, 
@import, @language, @propagate, @protected, @type, @version, or @vocab.
```

It would appear that we need to make sure examples and recommendations (at least if we want JSON-LD 1.1, which I suspect this is part of) must be maps.  

I've been running into this issue in some of my development work....
Comments and observations welcome..

### PR DESCRIPTION
**Pull request by mbjones**
Updated references to all contexts to load the context from `https://schema.org/` along with explaining how this results in using `http://schema.org/` as the namespace. Added new section to Getting Started to explain the nuances of `context` loading. Updated decision document to reflect these new decisions. Created a new SHACL shape to validate documents using these namespace decisions.

See discussion in issue #151, closes #151 . 

@datadavev and @fils could you please review the new text? And feel free to edit the branch directly. Thanks.

### CODE REVIEW

File: guides/Dataset.md [hunk 1]
```diff
@@ -391,7 +369,7 @@ NOTE: If you have a DOI, the citation text can be [automatically generated](http
 `schema:sameAs` is used here for the following reasons:
 
 1. It doesn't add too many more statements that might increase the page weight (which may impact major search engine crawlers stopping the crawl of schema.org markup).
-2. Crawlers that follow the URL for the short DOI can retrieve structured metadata for the DOI itself: 
+2. Crawlers that follow the URL for the short DOI can retrieve structured metadata for the DOI itself:
```
Comment (datadavev) [inline guides/Dataset.md:372]: It should be explained that such an operation does not retrieve the JSON-LD offered by the content source. Instead an alternative, less informative, and potentially out of date version is presented from CrossRef.

**Review by datadavev — APPROVED** [review]
Looks good. I have a minor beef with the example presenting short DOI resolution (which actually ends up with a CrossRef document, not one from the source), but that is a different issue.

File: guides/Dataset.md (hunk 1 above)
Comment (mbjones) [reply guides/Dataset.md:372]: Agreed. This section isn't the subject of this PR, which is focused on context retrieval, but I agree with your summary.

**Comment by mbjones** [comment]
@ashepherd This section modifies a lot of files and has large impacts on other work. I am inclined to merge it into develop quickly so that it doesn't interfere with other branches given that it is conflict free right now. If you or @fils have comments on the changes, could you let me know soonish? If changes are needed after the PR is merged, we can always do so. Thoughts?

**Comment by ashepherd** [comment]
Sounds good, go for it if you want to merge, and I can look later in the week?

**Pull request merged**